regress/iron-test-exec.sh

```diff
@@ -0,0 +1,419 @@
+#  Placed in the Public Domain.
+#
+#  Integration test runner for IronSSH tests. Based on the OpenSSH test-exec.sh
+#  script.
+
+
+#SUDO=sudo
+
+# Unbreak GNU head(1)
+_POSIX2_VERSION=199209
+export _POSIX2_VERSION
```

Comment:
No clue. Copied the file from `test-exec.sh` and pared it down. I didn't know about this so I figured no point in removing it.